data/saved_tasks.csv

```diff
@@ -0,0 +1,6 @@
+Uncategorised,#400,500,test,0
+Uncategorised,#200,100,test,0
+Uncategorised,#300,300,test,0
+Uncategorised,#100,900,test,0
+Uncategorised,#900,800,test,0
+
```

Comment:
are we or are we not supposed to be trackign the save file tho

Reply:
Shouldn't track save file. Ill update the .gitignore

Reply:
Done